tests/StackExchange.Redis.Tests/ScriptingTests.cs

```diff
@@ -107,7 +107,7 @@ public void HackyGetPerf()
         db.StringSet(key + "foo", "bar", flags: CommandFlags.FireAndForget);
         var result = (long)db.ScriptEvaluate(@"
 redis.call('psetex', KEYS[1], 60000, 'timing')
-for i = 1,5000 do
+for i = 1,500000 do
```

Comment:
@NickCraver What is the reason for changing the number of iterations for the loop?

Reply:
to be clear - only during lua script execution (catches up when the script exits)